### ISSUE TITLE
Integrate serviceworkers to allow auto refresh of project status

### ISSUE THREAD

**Issue by haroon-sheikh** [sitture/commit-status#9]

**Comment by hosseinnedaee** [comment]
Hi @haroon-sheikh 
does it solve the same issue as https://github.com/sitture/commit-status/issues/41 ?

**Comment by haroon-sheikh** [comment]
> Hi @haroon-sheikh
> does it solve the same issue as #41 ?

Yes, the main aim is to have the ability to auto-refresh with a toggle.

**Comment by hosseinnedaee** [comment]
Can I work on this?

**Comment by haroon-sheikh** [comment]
Yep go for it @hosseinnedaee

**Comment by hosseinnedaee** [comment]
Excusme @haroon-sheikh, what's the benefit using serviceworker over just set interval in the app?
We can use set interval to update status for example every 20 sec

**Comment by haroon-sheikh** [comment]
That’s fine. Let’s go ahead with using the interval and service worker can be done as a separate piece.
